guides/rails_guides/generator.rb

```diff
@@ -105,6 +106,10 @@ def select_only(guides)
         end
       end
 
+      def process_scss
+        system "sass ./assets/stylesrc/style.scss:./assets/stylesheets/style.css ./assets/stylesrc/highlight.scss:./assets/stylesheets/highlight.css"
+      end
```

Comment:
I think this is simple enough and should work for now.

I had a few minor suggestions that I pushed as a PR to your branch: https://github.com/jathayde/rails/pull/1